compact_test.go

```diff
@@ -312,44 +300,69 @@ func TestLeveledCompactor_plan(t *testing.T) {
 			},
 			expected: []string{"7", "8"},
 		},
-		// For overlapping blocks.
-		{
+		//|--------------|
+		//              |----------------|
+		//			         |--------------|
```

Comment:
not aligned properly.

Reply:
Sorry I don't quite understand...I already made some changes to the range values and the graphs to make them aligned in the previous commits. Can you give me an example?

Reply:
```
metaRange("1", 0, 20, nil),
metaRange("2", 19, 40, nil),
metaRange("3", 40, 60, nil),
```

the second block ends at 40 and the third block starts at 40 so the lines should be like
```
//|--------------|
//              |----------------|
//		             	         |--------------|
```

the same for the other places I commented.

Reply:
The last sample of "2" and the first sample of "3" overlap I guess? So isn't the representation right?